tests/compiler/number.ts

```diff
@@ -0,0 +1,7 @@
+import "allocator/arena";
+
+var a = 1;
+assert(a.toString() == "1");
+
+// TODO:
+// 1.toString()
```

Comment:
btw in js/ts this looks only like `1..toString()` but we have also integers so this should be legit in AS world

Reply:
Yeah, this is due to `1.` being parsed as a float, when followed by `toString` expects it to be an identifier, breaking the property access. Also happens in AS, but it appears that JS parsers have a specific syntax error there while AS emits `toString is not defined`.